biomage/rotate_ci/rotate_ci.py

```diff
@@ -249,10 +259,18 @@ def rotate_ci(token, org):
 
     g = Github(token)
     org = g.get_organization(org)
-    repos = org.get_repos()
+
+    if repo == "all":
+        repos = org.get_repos()
+    else:
+        try:
```

Comment:
doing a try with a generic exception and calling `quit()` is equivalent to do nothing, so I suggest to remove the try except and just allow the program to crash on that exception